<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>211</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="controlTab">
     <attribute name="title">
      <string>控制</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="detectButton">
        <property name="text">
         <string>开始检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="resultView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>resultView</zorder>
     <zorder>detectButton</zorder>
    </widget>
    <widget class="QWidget" name="ImageTab">
     <attribute name="title">
      <string>测试图</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QListWidget" name="correctImageListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="correctImageButtonLayout">
        <item>
         <widget class="QPushButton" name="importButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>导入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanListButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="batchDetectButton">
        <property name="text">
         <string>批量检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeImageButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>删除图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>371</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <widget class="QPushButton" name="closeButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;黑体&quot;;
background-color: red;
</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
